code-challenges.test.js

```diff
@@ -32,7 +33,54 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+describe("divisibleByThree", () => {
+  it("takes an object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+    expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+    expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+    expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+  })   
+})
+// Failing!
+//    FAIL  ./code-challenges.test.js
+//   divisibleByThree
+//   ✕ takes an object as an argument and decides if the number inside it is evenly divisible by three or not
+
+// ● divisibleByThree › takes an object as an argument and decides if the number inside it is evenly divisible by three or not
+
+//   ReferenceError: divisibleByThree is not defined
+
+// Pseudocode:
+// input: an object with a number
+// output: takes an object as an argument and decides if the number inside it is evenly divisible by three or not
+
+  // input: object1
+  // output: '15 is divisible by three'
+
+  // input: object2
+  // output: '0 is divisible by three'
+
+  // input: object3
+  // output: '-7 is not divisible by three
+
+  // Process:
+  // Create a function named divisibleByThree takes an object as an argument and decides if the number inside it is evenly divisible by three or not
+  // use modulo operator to determine if the object is evenly divisible by 3 or not by looking at the remainder 
+
+  // if there is no remainder then the number is evenly divisible by 3
+  // since there are two different outcomes, use two different conditions with separate outputs
+  // using string interpolation to create the statement that will reference the parameters to obtain the values of each number
+
+// FUNCTION
+
```

Comment:
Awesome job! I love the way you destructured the object and the pseudocode is very descriptive 💯